src/wix/WixToolset.Converters/WixConverter.cs

```diff
@@ -2854,6 +2854,45 @@ private static void RenameElementToStandardDirectory(XElement element)
             }
         }
 
+        // If TryGetInnerText succeeds, normalize the string value it returns by replacing all \r's \n's with spaces,
+        // then replacing all multiple spacws with single spaces. This is suitable for use in an XText representing a condition.
+        private static bool TryGetNormalizedInnerText(XElement element, out string value, out List<XNode> comments)
+        {
+            var found = TryGetInnerText(element, out var localValue, out comments);
+
+            if (found && null != localValue)
```

Comment:
Usually use `String.IsNullOrEmpty` to check for empty string instead of `null`, unless there is something very special about null. So, `!String.IsNullOrEmpty(localValue)` would be better here.

Reply:
TryGetInnerText() returns null when there is no inner text, but it certainly won't do any harm to treat empty strings the same way I treat nulls. I'll fix it.